qemu/tests/cfg/edk2_basic.cfg

```diff
@@ -0,0 +1,34 @@
+- edk2_basic:
+    virt_test_type = qemu
+    only Linux
+    only x86_64, aarch64
+    type = edk2_basic
+    start_vm = no
+    login_timeout = 240
+    reboot_count = 5
+    check_messgae = "enter to boot the selected OS"
+    skip_image_processing = yes
+    line_numbers = 40
+    variants:
+        - @default:
+        - iommu_test:
+            virtio_dev_iommu_platform = on
+            virtio_dev_filter = '^(?:(?:virtio-)|(?:vhost-))(?!(?:balloon)|(?:user)|(?:iommu))'
+            variants:
+                - intel_iommu:
+                    only q35
+                    only HostCpuVendor.intel
+                    machine_type_extra_params = "kernel-irqchip=split"
+                    virtio_dev_ats = on
+                    virtio_dev_aer = on
+                    intel_iommu = yes
+                    enable_guest_iommu = yes
+                - smmuv3:
+                    only aarch64
+                    machine_type_extra_params += ",iommu=smmuv3"
+                - viommu:
+                    only aarch64 q35
```

Comment:
EDK2 means OVMF in x86 and only supports q35, so this limitation is unnecessary if you have the correct definition on the top.

Reply:
Thanks review, updated